packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-rest/actual.js

```diff
@@ -0,0 +1,15 @@
+const a = 1;
+function rest(b = a, ...a) {
```

Comment:
True, I also checked Edge 13 with default function parameters activated. It throws "Use before define" Error. So now we have three ways to choose from :D